### ISSUE TITLE
Bring NAPS2 to Flathub.org

### ISSUE THREAD

**Issue by MG-LSJ** [cyanfish/naps2#206]
Though NAPS2 is already available in flatpak format, it would be great if you release it on flathub.org, as its the default source for many linux distro app stores.

**Comment by ivdimitro** [comment]
In theory the submission to flathub can be made from other not strictly the app writer/owner [process](https://docs.flathub.org/docs/for-app-authors/submission/)
@cyanfish will this be something you are OK with or publish to flathub is in your plans anyway?

**Comment by cyanfish** [comment]
Yes, I do plan to eventually publish on Flathub (along with the Windows and Mac stores too).

**Comment by RokeJulianLockhart** [comment]
> https://github.com/cyanfish/naps2/issues/206#issuecomment-1818023079

https://github.com/microsoft/winget-pkgs/issues/90081#issuecomment-2015496123

**Comment by rambo919** [comment]
It's on Flathub now but it has driver permission issues, this can be remedied with something like Flatseal but I don't know which option to use.

**Comment by repires** [comment]
> It's on Flathub now but it has driver permission issues, this can be remedied with something like Flatseal but I don't know which option to use.

Where is the link? I didn´t find it. @cyanfish any updates on the publishing? I created a thread on the forum.

https://discourse.flathub.org/t/naps2-not-another-pdf-scanner/6493/2

**Comment by Hoeze** [comment]
any updates on this? I would like to have automatic updates for my flatpaks

**Comment by repires** [comment]
Do you a third party to add your flatpak?
@cyanfish

**Comment by muelli** [comment]
The manifest seems to be [here](https://github.com/cyanfish/naps2/commit/dcb1e045d1fd94783fff8e4193abc678d506baf6#diff-3c922985e077775c2b7641859869a01e2eb0b83d1e9b478d01d0d693fcf1a5e4) in case somebody wants to propose it for Flathub.

**Comment by repires** [comment]
Hi @cyanfish . The windows store was added. Are you able to publish on the flathub repo too?

**Comment by pabloab** [comment]
Not sure if NAPS2 is abandoned or using old dependencies, but [from above thread](https://discourse.flathub.org/t/naps2-not-another-pdf-scanner/6493/4):

> [Requirements | Flathub Documentation](https://docs.flathub.org/docs/for-app-authors/requirements#end-of-life-dependency-policy) says, that applications using an EOL runtime won’t be accepted and Naps2 doesn’t seem to be updated anymore.

**Comment by MajorTomDE** [comment]
+1 for publishing on Flathub
